src/model/deviceInfo.ts

```diff
@@ -0,0 +1,21 @@
+export interface DeviceInfoParam {
+    key: string;
+    value: string;
+}
+
+export class DeviceInfo {
+    params: Array<DeviceInfoParam>;
+
+    public getParam(key: string) {
```

Comment:
Simplify this method by failing fast instead of nesting conditionals

Reply:
Created https://www.pivotaltracker.com/story/show/171322089 to address non-essential code improvements so we can get the security fix out asap